.babelrc

```diff
@@ -1,21 +1,27 @@
 {
-	"presets": [
-		"@wordpress/default"
-	],
+	"presets": [ "@wordpress/default" ],
 	"plugins": [
-		"transform-async-generator-functions"
+		"@babel/plugin-transform-async-to-generator",
+		"@wordpress/babel-plugin-import-jsx-pragma",
+		[
+			"@babel/plugin-transform-react-jsx",
+			{
+				"pragma": "wp.element.createElement",
```

Comment:
This in particular feels unnecessary, but I kept getting `createElement is not defined` errors otherwise